libs/mark-flow-ui/src/components/candidate_contest.tsx

```diff
@@ -103,31 +103,31 @@ export function CandidateContest({
     }
   }, [deselectedCandidate]);
 
-  function addCandidateToVote(id: string) {
+  function addCandidateToVote(id: string, event: SyntheticEvent) {
```

Comment:
Feels a little odd to me to be passing this event around, but I do see why. If the only reason you need it is to determine the update method, maybe make that explicit here?

```ts
enum VoteUpdateMethod {
  Touch = 'touch',
  Other = 'other',
}

function addCandidateToVote(id: string, method: VoteUpdateMethod) {
```

Reply:
Added in [8f2df03](https://github.com/votingworks/vxsuite/pull/4284/commits/8f2df03b9d6dd25b17789d68620c13a21987ec42). I like how this makes it more clear why we need to pass a value up and keeps the `UpdateVoteFunction` type simple. The downside is conversion from `Event` to `VoteUpdateInteractionMethod` isn't centralized. Overall felt like the best approach but open to suggestions.